packages/settings-library/src/settings_library/redis.py

```diff
@@ -33,7 +33,7 @@ class RedisSettings(BaseCustomSettings):
     def build_redis_dsn(self, db_index: RedisDatabase) -> str:
         return str(
             RedisDsn.build(  # pylint: disable=no-member
-                scheme="redis",
+                scheme="rediss" if self.REDIS_SECURE else "redis",
```

Comment:
MINOR: don't you think this is more readable?  
```python
scheme= "redis" + "s" if self.REDIS_SECURE else ""
``` 
since it stresses what changes. With your approach i did not see the change at first :-)

Reply:
If we talk about readability, this would be the best:

```python
if self.REDIS_SECURE:
    redis_protocol = "rediss"
else:
    redis_protocol = "redis"
```